src/main/java/src/chapter5/question1/Question1.java

```diff
@@ -0,0 +1,14 @@
+package src.chapter5.question1;
+
+public class Question1 {
+    public static boolean isInternedString(String string) {
+        return string == string.substring(0);
+    }
+
+    public static boolean isStringBuilderSubstringInterned() {
```

Comment:
Receive the string to be checked as a parameter.

Reply:
Fixed.